.github/workflows/smoke-test-daily.yml

```diff
@@ -1,7 +1,7 @@
 name: Smoke test daily
 on:
-    # schedule:
-    #     - cron: '25 3 * * *'
+    schedule:
+        - cron: '25 3 * * *'
```

Comment:
Enabled the cron  schedule for this workflow to start running all E2E tests except for the shopper ones.